client/apiversion.go

```diff
@@ -208,27 +240,23 @@ func (c *authenticatingClient) getAPIVersions(serviceCatalogURL string) (*apiURL
 		rootURL:          *url,
 		serviceURLSuffix: strings.Join(pathParts, "/"),
 	}
-	if err := c.sendRequest("GET", baseURL, c.Token(), requestData); err != nil {
-		if gooseerrors.IsNotFound(err) {
-			// Some OpenStack clouds do not support the
-			// version endpoint, and will return a status
-			// code of 404. We check for 404 and return an
-			// empty set of versions.
-			logger.Warningf("API version discovery failed: %v", err)
-			c.apiURLVersions[baseURL] = apiURLVersionInfo
-			return apiURLVersionInfo, nil
-		}
-		return nil, err
+	if err := c.sendRequest("GET", getVersionURL, c.Token(), requestData); err != nil {
+		logger.Debugf("API version discovery failed: %v", err)
```

Comment:
This was warn -- I think we should keep it on warn. We would want every user to see this.

Reply:
done